City-of-Helsinki/drupal-hdbt
# UHF-5573: Adjusted news item spacing and teaser scaling

# Adjust spacing on news item listings [UHF-5573](https://helsinkisolutionoffice.atlassian.net/browse/UHF-5573)
Discussed with designers about the news item listings and applied their requests on the spacing of the news items.

## What was done
* Changed news item teaser spacing between image and title to 24px and switched the news items without images to have 100% width on mobile sizes.

## How to install
* Test this with frontpage-instance.
* Make sure your instance is up and running on latest dev branch.
    * `git pull origin dev`
    * `make fresh`
* Update the HDBT theme
    * `composer require drupal/hdbt:dev-UHF-5573_spacing_adjustments_to_news_listing`
* Run `make drush-cr`

## How to test
* Go to https://helfi-etusivu.docker.so/fi/news and make sure that the spacing between title and image is 24px and that the news item teasers without images will scale to 100% width when the browser is smaller than 768px.
